src/FilterSelect.js

```diff
@@ -0,0 +1,30 @@
+// import Button from '@material-ui/core/Button';
+// import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import Select from '@material-ui/core/Select';
+import React, { Component } from 'react';
+import {FILTER_TYPE} from './WaitingRoom.js'
+
+export default class FilterSelect extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { filterType: FILTER_TYPE.MY_GAMES };
```

Comment:
Does this make sense to keep track of here and in the waiting room component? Or should it just be stored in the waiting room component and passed down as a prop?

Reply:
Let's do the latter. Storing the state just in `<WaitingRoom>` feels right and just pass it down to `<FilterSelect>` as a prop.

Reply:
Sounds good!